_data/roadmap.yml

```diff
@@ -1,60 +1,60 @@
 - year: 2014
   accomplishments:
     - name: Launched on Bitcointalk
-      date: April 18, 2014
+      date: 18 April 2014
```

Comment:
Care to point out where? A simple grep of monero-site from 2014-2017 shows that this is not the case. In fact, the majority of dates are labeled YYYY-MM-DD.